server/models/User.model.ts

```diff
@@ -85,4 +109,8 @@ User.init({
   },
 });
 
+User.associations = {
+  flags: User.hasMany(Flag, { foreignKey: 'user_id' }),
+};
+
```

Comment:
What is this bit of code doing? Isn't the `hasMany` association already defined within the static `associate` method?

I couldn't find in the docs where associations are defined like this